coordinator/core/storewrapper.go

```diff
@@ -34,6 +34,7 @@ type storeWrapper struct {
 		Get(string) ([]byte, error)
 		Put(string, []byte) error
 	}
+	metrics *StoreWrapperMetrics // can be nil
```

Comment:
```suggestion
```
storeWrapper is supposed to be completely stateless. I think callers of putState should update the metric. There are actually only two relevant callers.